Makefile.toml

```diff
@@ -0,0 +1,107 @@
+[env]
+CARGO_MAKE_EXTEND_WORKSPACE_MAKEFILE = "true"
+CARGO_MAKE_WORKSPACE_EMULATION = true
+CARGO_MAKE_CRATE_WORKSPACE_MEMBERS = [
+	"crt0stack",
+	"enarx-keep-sgx",
+	"enarx-keep-sgx-shim",
+	"enumerate",
+	"intel-types",
+	"iocuddle",
+	"iocuddle-sgx",
+	"memory",
+	"nolibc",
+	"payload",
+	"sev",
+	"sev-show",
+	"sev-types",
+	"sgx-crypto",
+	"sgx-show",
+	"sgx-types",
+	"span",
+	"testing",
+	"units",
+	"vdso",
+	"vmsyscall"
+]
+
+[tasks.audit]
+install_crate = "cargo-audit"
+command = "cargo"
+args = ["audit"]
+
+[tasks.deny]
+install_crate = "cargo-deny"
+command = "cargo"
+args = ["deny", "check", "licenses"]
+
+[tasks.test]
+command = "cargo"
+args = ["test"]
```

Comment:
Is this overriding the default `tasks.test`? If so why? Do you need to strip out the additional functionality for some reason?

https://github.com/sagiegurari/cargo-make/blob/master/src/lib/descriptor/makefiles/rust.toml#L375

Reply:
Ah true. The default `test` target turns on all features for the crate, which I wanted to avoid the `dangerous_tests` features for SEV. However, I can remove this from the root Makefile and specify not to use dangerous_tests by default in the `sev` crate Makefile.